src/odemis/acq/stream/_projection.py

```diff
@@ -67,6 +67,7 @@ def projectAsRaw(self):
 
         returns: None if failure, or a DataArray
         """
+        raise NotImplemented()
```

Comment:
Drop the `return` line, as it's never reached.

Reply:
Ah oops - I left this in from debugging when I was testing to see which projectAsRaw got called.